scripts/polynomial_reference_check.rb

```
@@ -0,0 +1,184 @@
+#!/usr/bin/env ruby
```

RuboCop (1.85.1)

[warning] Script file polynomial_reference_check.rb doesn't have execute permission.

(Lint/ScriptPermission)